draft-ietf-aipref-vocab.md

```diff
@@ -315,11 +315,17 @@ The use of assets for Search is a proper subset of Automated Processing usage.
 
 ## Vocabulary Extensions {#vocab-extension}
 
-Systems referencing the vocabulary MUST NOT introduce additional categories
+Extensions to this vocabulary need to be defined in an RFC
+that updates this document.
+
+Any future extensions to this vocabulary MUST NOT introduce additional categories
 that include existing categories defined in the vocabulary.
 That is, new categories of use can be defined as a subset of an existing category,
 but not a superset.
 
+Systems that use this vocabulary might define their own extensions,
+see {{mapping}} for how those are managed.
```

Comment:
Hmm. That refers to an alternative serialisation, which isn't really what this is about -- it's about an external consumer of the vocabulary that repurposes or augments it outside the scope of this specification. Something can do that without using the alternative serialisation (which is more about attachment than the vocabulary).